rusk/Cargo.toml

```diff
@@ -21,33 +21,34 @@ path = "src/bin/main.rs"
 
 [dependencies]
 tonic = "0.6"
-tokio = { version = "1.12", features = ["rt-multi-thread", "time", "fs", "macros"] }
+tokio = { version = "1.15", features = ["rt-multi-thread", "time", "fs", "macros"] }
 async-stream = "0.3"
-dusk-poseidon = { version = "0.23.0-rc", features = ["canon"] }
-dusk-plonk = { version = "0.9", features = ["canon"] }
-dusk-bls12_381 = { version = "0.8", features = ["canon"] }
-dusk-jubjub = { version = "0.10", features = ["canon"] }
-dusk-pki = "0.9.0-rc"
-phoenix-core = "0.15.0-rc"
-dusk-schnorr = "0.9.0-rc"
 tracing = "0.1"
-tracing-subscriber = "0.2"
-clap = "2.33.3"
+tracing-subscriber = "0.3"
+clap = "2.33"
```

Comment:
We might want to try to update to `clap` 3.0 and see if it's break anything. If it does, we can revert to `2.33` for the time being.